bool read_status
bool write_status
int16 max_distance
bool persistent_error
bool interference
bool temperature_error
bool temporary_error
bool voltage_error
int8 sensor_id
int8 sort_index
int8 radar_power_cfg
bool ctrl_relay_cfg
int8 output_type_cfg
bool send_quality_cfg
bool send_ext_info_cfg
int8 motion_rx_state
bool rcs_threshold
